src/input/hotkey.h

```diff
@@ -17,6 +17,9 @@ void hotkey_esc(void);
 void hotkey_page_up(void);
 void hotkey_page_down(void);
 void hotkey_enter(void);
+void hotkey_new_map(void);
```

Comment:
These functions are only used within `hotkey.c`, so they do not need to be exposed to other code.

Reply:
Makes sense, I've just followed the other ones there.
So I guess those should be changed too? (Although in different PR imho)